doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-requests-proc.adoc

```diff
@@ -0,0 +1,108 @@
+[id='security-management-rest-api-requests-proc_{context}']
+= Sending requests with the Security Management REST API using a REST client or curl utility
+
+The Security Management REST API enables you to manage the groups, roles, and users in {PRODUCT} without using the {CENTRAL} user interface. You can send Security Management REST API requests using any REST client or curl utility.
+
+.Prerequisites
+* {CENTRAL} is installed and running.
+* You have `admin` user role access to {CENTRAL}.
+
+.Procedure
+. Identify the relevant xref:security-management-rest-api-endpoints-ref_kie-apis[API endpoint] to which you want to send a request, such as `[GET] /groups` to retrieve groups in {CENTRAL}.
+. In a REST client or curl utility, enter the following components for a `GET` request to `/groups`. Adjust any request details according to your use case.
++
+--
+For REST client:
+
+* *Authentication*: Enter the user name and password of the {CENTRAL} user with the `admin` role.
+* *HTTP Headers*: Set the following header:
+** `Accept`: `application/json`
+* *HTTP method*: Set to `GET`.
+* *URL*: Enter the Security Management REST API base URL and endpoint, such as `\http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/groups`.
+
+
+For curl utility:
+
+* `-u`: Enter the user name and password of the {CENTRAL} user with the `admin` role.
+* `-H`: Set the following header:
+** `accept`: `application/json`
```

Comment:
Probably would be good to unify headers to start with upper case for both cases (REST client and curl utility).

Reply:
Done